deployments/torpedo-k8s-ssh.yaml

```diff
@@ -51,7 +51,14 @@ spec:
   - name: torpedo
     image: harshpx/torpedo:latest
     #command: ["./torpedo", "fire", "--scheduler", "k8s", "--storage", "pxd", "--node-driver", "ssh" ]
-    command: ["./torpedo", "fire", "--scheduler", "k8s", "--storage", "pxd", "--node-driver", "ssh", "--tests", "testSetupTearDown" ]
+    command: ["./torpedo", "fire", "--scheduler", "k8s", "--storage", "pxd", "--node-driver", "ssh", "--tests", "testSetupTearDown", "--log-location", "/mnt/tsb" ]
```

Comment:
should we keep this file consistent with the default log location we have above?